components/meta-packages/xorg-video/xorg-video.p5m

```diff
@@ -33,4 +33,5 @@ depend fmri=pkg:/x11/server/xorg/driver/xorg-video-r128 type=require
 depend fmri=pkg:/x11/server/xorg/driver/xorg-video-savage type=require
 depend fmri=pkg:/x11/server/xorg/driver/xorg-video-trident type=require
 depend fmri=pkg:/x11/server/xorg/driver/xorg-video-vesa type=require
+depend fmri=pkg:/x11/server/xorg/driver/xorg-video-vboxvideo type=require
```

Comment:
Please, bump COMPONENT_REVISION for this component.

Reply:
Done.